src/components/cursor-controller.js

```diff
@@ -312,7 +312,7 @@ AFRAME.registerComponent("cursor-controller", {
       !this.isMobile ||
       this.hasPointingDevice ||
       !this.activeTouch ||
-      e.touches.some(touch => touch.identifier === this.activeTouch.identifier)
+      this.some(e.touches, touch => touch.identifier === this.activeTouch.identifier)
```

Comment:
is the `this.some` helper because `e.touches` is a `TouchList` and not an `Array`? could you remove the `some` method and call directly `Array.prototype.some.call(e.touches, touch => touch.identifier === this.activeTouch.identifier)`?

Reply:
yes